contribs/gmf/src/services/themesservice.js

```diff
@@ -54,13 +58,13 @@ gmf.Themes = function($http, treeUrl, isThemePrivateUrl) {
    * @type {string}
    * @private
    */
-  this.treeUrl_ = treeUrl;
+  this.treeUrl_ = '';
 
   /**
    * @type {string}
    * @private
    */
-  this.isThemePrivateUrl_ = isThemePrivateUrl;
+  this.isThemePrivateUrl_ = '';
```

Comment:
Okay... then I pass the URL as parameter on function 'isThemePrivate' ? Luxembourg project will must be adapted.